frontend/components/Header/Nav/Pillars/pillars.ts

```diff
@@ -0,0 +1,27 @@
+import { PillarColours, palette } from '@guardian/pasteup/palette';
+
+export const pillarNames: Pillar[] = [
+    'news',
+    'opinion',
+    'sport',
+    'culture',
+    'lifestyle',
+];
+
+export const pillarPalette: { [K in Pillar]: PillarColours } = {
+    news: palette.news,
+    opinion: palette.opinion,
+    sport: palette.sport,
+    culture: palette.culture,
+    lifestyle: palette.lifestyle,
+};
```

Comment:
Why not keep these in `pasteup`?

Reply:
I thought about this, but I think the modelling of pillars doesn't belong in pasteup.  

The pillars in pasteup are defined by the <a href="https://design.theguardian.com/#colour-palette-4">design guide.</a>

Whereas the pillars in pillars.ts are defined by our split of content into pillars. My aim for this is to only ever check once whether a given value of pillar is actually a pillar by having every pillar reference use these types. I probably should move this file to somewhere more sensible than the nav though.

Reply:
Sounds good, thanks for the explanation. Shall we move this up to the top level `frontend` directory for now?

Reply:
Moved. 👍